src/components/routerView.browser.spec.ts

```diff
@@ -70,21 +66,23 @@ test('renders components for initial route', async () => {
 })
 
 test('updates components when route changes', async () => {
-  const routes = createRoutes([
-    {
+  const routes = [
+    createRoute({
       name: 'childA',
       path: '/childA',
       component: { template: 'ChildA' },
-    }, {
+    }),
+    createRoute({
       name: 'childB',
       path: '/childB',
       component: { template: 'ChildB' },
-    }, {
+    }),
+    createRoute({
       name: 'childC',
       path: '/childC',
       component: { template: 'ChildC' },
-    },
-  ])
+    }),
+  ] as const
```

Comment:
Otherwise its just `Route[]`? 

Random note, the route names don't really make sense in this test.

Reply:
as const wasn't actually needed, at some point I added thinking it was the issue but it wasn't. Removed everywhere.

not sure what you don't like about route names but I changed them, lemme know if that's not better